test/uk/gov/hmrc/gform/sharedmodel/formtemplate/generators/ComponentTypeGen.scala

```diff
@@ -24,6 +24,7 @@ import uk.gov.hmrc.gform.sharedmodel.formtemplate._
 trait ComponentTypeGen {
 
   def displayWidthGen: Gen[DisplayWidth] = Gen.oneOf(DisplayWidth.values.toSeq)
+  def rowsGen: Gen[Option[Int]] = Gen.option(Gen.chooseNum(5, 50))
```

Comment:
For test purpose, we dont need to restrict to 5-50. Any positive number will do.

Reply:
Changed to **posNum**.

Reply:
the value generated is irrelevant, as long its a positive number